docs/data/material/components/snackbars/snackbars.md

```diff
@@ -18,8 +18,6 @@ They may contain a text action, but no icons. You can use them to display notifi
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
 
-**Frequency**: Only one snackbar may be displayed at a time.
```

Comment:
This is still the recommendation of Material Design, proof: https://material.io/components/snackbars#usage. We could make it clearer, but I think that the message should still be present.

Reply:
@danilo-leal Can you take a look?